routesrv/routesrv_test.go

```diff
@@ -69,8 +69,9 @@ func loadKubeYAML(t *testing.T, path string) io.Reader {
 
 func newRouteServer(t *testing.T, kubeServer *httptest.Server) *routesrv.RouteServer {
 	return newRouteServerWithOptions(t, routesrv.Options{
-		SourcePollTimeout: pollInterval,
-		KubernetesURL:     kubeServer.URL,
+		SourcePollTimeout:                      pollInterval,
+		KubernetesURL:                          kubeServer.URL,
+		KubernetesDefaultLoadBalancerAlgorithm: "roundRobin",
```

Comment:
this should reference a const instead of using `"roundRobin"`. You need to expose the const, but this sounds ok.